ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-4.10.yaml

```diff
@@ -152,6 +152,19 @@ tests:
     - chain: upi-vsphere-pre
     - ref: cucushift-pre
     workflow: cucushift-e2e
+- as: e2e-azure-cucushift-upi
+  interval: 24h
+  steps:
+    cluster_profile: azure4
+    env:
+      E2E_RUN_TAGS: '@azure-upi'
+      TAG_VERSION: '@4.10'
+    post:
+    - chain: upi-azurestack-post
+    pre:
+    - chain: upi-azurestack-pre
```

Comment:
What's the diff between azurestack and azure ?

Reply:
@dis016 I found a page about cluster_profile, maybe you can try  ClusterProfile = "azurestack"?
https://github.com/openshift/ci-tools/blob/master/pkg/api/types.go#L1073

Reply:
upi on azure is based on azure stack https://azure.microsoft.com/en-us/products/azure-stack/hub/